docs/LibcSupport.md

```diff
@@ -18,7 +18,8 @@ stdio.h | Partial | All I/O functions implicitly call out to untrusted host. Sup
 stdlib.h | Partial | Unsupported functions: div(), imaxabs(), imaxdiv(), ldiv(), lldiv() |
 string.h | Partial | Unsupported functions: strerror(), strsignal() |
 tgmath.h | Partial | Unsupported functions: acosh(), asinh(), fmal(), lgamma(), lgamma_r(), scalbn(), scalbnf(), scalbnl(), sinh(), sinhl(), tgamma() |
-threads.h | Partial | Synchronization primitives are not secure across calls to host. Threads are still scheduled by the untrusted host process and an enclave cannot rely on threads making forward progress. Unsupported functions: pthread_create(), pthread_join() and pthread_detach() |
+pthread.h | Partial | Synchronization primitives are not secure across calls to host. Threads are still scheduled by the untrusted host process and an enclave cannot rely on threads making forward progress. Unsupported functions: pthread_create(), pthread_join() and pthread_detach() |
```

Comment:
pthread.h has a larger API specification: https://pubs.opengroup.org/onlinepubs/7908799/xsh/pthread.h.html
The unsupported list is larger than mentioned here.

Reply:
@anakrish Agreed, thanks for the catch.